oak_functions/examples/metrics/client/rust/src/main.rs

```diff
@@ -19,7 +19,7 @@
 use anyhow::Context;
 use clap::Parser;
 use oak_functions_abi::proto::Request;
-use oak_functions_client::{Client, Session};
+use oak_functions_client::{Session, UnaryClient as Client};
```

Comment:
Would it be possible to avoid renaming the import? It's rather unusual (in my experience, at least), unless it's actually necessary to avoid a collision.

Reply:
Not strictly needed, removed